upstairs/src/client.rs

```diff
@@ -414,7 +413,7 @@ impl DownstairsClient {
         job
     }
 
-    /// Ensures that the given job is in the job queue and in `IOState::New`
+    /// Ensures that the given job is in the job queue
     pub(crate) fn replay_job(&mut self, job: &mut DownstairsIO) {
         // If the job is InProgress, then we can just go back to New and no
```

Comment:
The comment here about state `New` is no longer correct.

Reply:
fixed!